keyboard_rewrite/src/lib/y2_keyboard/keyboard_layout.rb

```diff
@@ -18,8 +18,8 @@ def initialize(code, description)
 
     def self.all
       raw_layouts = Cheetah.run("localectl", "list-keymaps", stdout: :capture)
-      layout_codes = raw_layouts.lines.map { |string| string.split.first }
-      codes_with_description = layout_codes.select { |code| LAYOUT_CODE_DESCRIPTIONS.key?(code) }
+      codes = raw_layouts.lines.map(&:strip)
+      codes_with_description = codes.select { |code| LAYOUT_CODE_DESCRIPTIONS.key?(code) }
```

Comment:
np: I prefer to use single letter variables when using one-line blocks. See [our Ruby style guide](https://github.com/SUSE/style-guides/blob/master/Ruby.md#parameters).